package com.fiap.hackaton.entity.coletaInsumo.exception;

public class ColetaInsumoNotFoundException extends Exception {

    public ColetaInsumoNotFoundException() {
        super("A coleta n√£o foi encontrada!");
    }
}
